src/classes/BDI_ManageAdvancedMappingCtrl.cls

```diff
@@ -458,4 +527,126 @@ public class BDI_ManageAdvancedMappingCtrl {
         }
     }
 
-}// BDI_ManageAdvancedMappingCtrl
+    public class DataImportObjectMapping {
+        @AuraEnabled public String DeveloperName;
+        @AuraEnabled public String MasterLabel;
+        @AuraEnabled public String Custom_Mapping_Logic_Class;
```

Comment:
Yes, I'm doing it differently here because I think its cleaner to not have these prefixes/suffixes everywhere.  There is no requirement to directly deserialize into the object, and in fact that limits what additional fields you can add to your wrapper for other purposes.

Reply:
@callen-sfdo I think that makes sense! Do you think it's worth trying to bring the two into sync and try to standardize our process across the two?